module/project/templates/helper.js

```diff
@@ -1079,6 +1079,18 @@ module.exports = class ProjectTemplatesHelper {
                         userId : userId,
                         projectTemplateId : templateData[0]._id
                     },["_id"]);
+
+                    //check for program enrollment if enrolled then add programJoined true
+                    let programJoined = await programUsers.programUsersDocument({userId: userId},["programId"])
+                    project.map((projects)=>{
```

Comment:
@ankitpws can the logic be that for all projects of programs that have requestForPIIConsent true programJoined will be true always